cmd/openshift-install/agent_internal_integration_test.go

```diff
@@ -84,6 +84,8 @@ func runIntegrationTest(t *testing.T, testFolder string) {
 
 	testscript.Run(t, testscript.Params{
 		Dir: testFolder,
+		// Uncomment below line to help debug the testcases
+		// TestWork: true,
```

Comment:
Not really sure what this does...

Reply:
It helps to save the temp work directory thereby one can debug the files and other directories in the work dir.